src/docs/admin/references/NodeSourceInfrastructureReference.adoc

```diff
@@ -948,6 +948,92 @@ It is advised to embed credentials and certificates in the configuation file, in
 
 - *image:* (Optional) Docker image used to deploy ProActive Nodes. When it is not set, its default value `java:8-jre` is used.
 
+- *nodeDeploymentTemplateFile:* (Optional) Kubernetes deployment template file for deploying ProActive node pods. This template file tells Kubernetes platform the desired running environment for the ProActive node. Each ProActive node will use this template file to create its specific Kubernetes deployment. 
+The default deployment template file creates a running environment with both ProActive node and a Docker server, so that the tasks running on the Kubernetes nodes can execute Docker commands. It is supported through the Docker-in-Docker approach (i.e., running a Docker daemon inside the Kubernetes host container system). Specifically, it creates one pod with two containers (`proactive-node` and `dind-daemon`). 
```

Comment:
The default **Kubernetes** template file creates **an environment (i.e., a pod)** where both a ProActive node and a Docker **engine** are started. This way, the tasks executed by the ProActive node can also run Docker commands. **Such an environment is supported through** a Docker-in-Docker approach (i.e., running a Docker daemon inside the Kubernetes host container system). Specifically, this environment consists of one pod with two containers (`proactive-node` and `dind-daemon`).